clients/factory.go

```diff
@@ -244,7 +262,7 @@ func (f factory) CreateClassicClient() (*rest.Client, error) {
 		return nil, ErrClassicURLMissing
 	}
 
-	return f.createRestClient(f.classicURL, auth.NewTokenBasedClient(f.accessToken))
+	return f.createRestClient(f.classicURL, auth.NewApiTokenBasedClient(context.TODO(), f.accessToken))
```

Comment:
Is there a follow-up for this one or should we now add the context to the parameters?

Reply:
Adding it is a breaking change, that's why I didn't do it. 
There are for sure possible follow ups, but it's not urgent at all. In fact, the context is not used at all, `nil` would have been a valid value. But sonar shouted at me :)